backend/samfundet/views.py

```diff
@@ -760,6 +761,60 @@ def get_queryset(self) -> Response:
         return Recruitment.objects.filter(visible_from__lte=timezone.now(), actual_application_deadline__gte=timezone.now())
 
 
+class DownloadRecruitmentAdmissionGangCSV(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(
+        self,
+        request: Request,
+        recruitment_id: int,
+        gang_id: int,
+    ) -> HttpResponse:
+        recruitment = get_object_or_404(Recruitment, id=recruitment_id)
+        gang = get_object_or_404(Gang, id=gang_id)
+        admissions = RecruitmentAdmission.objects.filter(recruitment_position__gang=gang, recruitment=recruitment)
+
+        response = HttpResponse(
+            content_type='text/csv',
+            headers={'Content-Disposition': 'attachment; filename="somefilename.csv"'},
```

Comment:
Og importer datetime

```suggestion
        filename = f"opptak_{gang.name_nb}_{datetime.now().strftime('%Y-%m-%d %H:%M')}.csv"
        response = HttpResponse(
            content_type='text/csv',
            headers={'Content-Disposition': f'Attachment; filename="{filename}"'},
```

Reply:
Done